README.md

```diff
@@ -348,15 +329,6 @@ nvm install node
 NVM_NODEJS_ORG_MIRROR=https://nodejs.org/dist nvm install 4.2
 ```
 
-To use a mirror of the io.js binaries, set `$NVM_IOJS_ORG_MIRROR`:
```

Comment:
When in a situation where you’re overriding the node mirror, this one still needs to resolve to something or else all of nvm’s remote commands will break.

Reply:
No, why i mean is, `nvm ls-remote` will fail unless it can reach both nodejs.org and iojs.org; and if one is using the node mirror env var, it’s probably because nodejs.org can’t be reached - so that implies iojs.org can’t be reached either, which means this env var still needs to be set to something.